### ISSUE TITLE
Alter Ego

### ISSUE THREAD

**Issue by Pbm13** [Monika-After-Story/MonikaModDev#5698]
It seems kinda dumb, but would you consider putting a reference to a game called Alter Ego? its a game by Caramel Column and I thought it would be kinda neat for Monica to adopt a hairstyle or suit similar to the one Es, "your alter ego", has. just a thought :/

![es](https://user-images.githubusercontent.com/36735471/80897421-52df4a80-8cc6-11ea-8c1a-0e9f05a4f62a.png)

**Comment by lunulae** [comment]
I do think this would be a very thematic cosplay idea for the future. Moni's long hair in that style of braid would look very cool as well, I think.
I've actually considered the idea of writing a topic about this game, but it seems like one that might need to be handled very carefully if at all.

**Comment by Rai99** [comment]
It would most likely have to be a spritepack. 

Alter Ego was brought up in #5478, but for the purposes of book discussion.

**Comment by Pbm13** [comment]
yeah, lunulae has a point since Caramel Column is poised to release another ES centered game in early to mid June, I see why it might prove a problem. However the hairstyle seems possible, if anything and the literature discussions

**Comment by ThePotatoGuy** [comment]
added to user art requests queue
